CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/usecase/GetUsersUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.domain.usecase
+
+import co.nimblehq.domain.entity.UserEntity
+import co.nimblehq.domain.repository.UserRepository
+import javax.inject.Inject
+
+class GetUsersUseCase @Inject constructor(private val userRepository: UserRepository) {
+    suspend fun execute(page: Int, size: Int): UseCaseResult<List<UserEntity>> {
```

Comment:
Can we have a line-break before this line?

Reply:
@lydiasama 👌